homeassistant/components/monzo/api.py

```diff
@@ -0,0 +1,26 @@
+"""API for Monzo bound to Home Assistant OAuth."""
+
+from aiohttp import ClientSession
+from monzopy import AbstractMonzoApi
+
+from homeassistant.helpers import config_entry_oauth2_flow
+
+
+class AsyncConfigEntryAuth(AbstractMonzoApi):
```

Comment:
The class name is a bit weird IMHO

Reply:
The new integration scaffold creates the class and a couple dozen integrations including a couple I was using for reference have kept the name. I agree it's a bit weird though, I'll change it